samples/CommunityToolkit.Maui.Sample/Pages/Views/Expander/ExpanderPageCS.cs

```diff
@@ -28,11 +29,16 @@ public ExpanderPageCS()
 				new Expander
 				{
 					Header = new Label()
-								.Text("Expander (Tap Me)")
+								.Text("Expander\n(Tap Me)")
+								.TextCenter()
 								.Font(bold: true, size: 18),
 
 					Content = new VerticalStackLayout()
 					{
+						new Button()
```

Comment:
Let's update sample to be more MAUI related: https://github.com/CommunityToolkit/Maui/pull/1219/files